charts/agent/templates/_config-exporters.tpl

```diff
@@ -15,7 +15,7 @@ otlphttp/observe/base:
 
 {{- define "config.exporters.otlphttp.observe.entity" -}}
 otlphttp/observe/entity:
-    endpoint: "{{ .Values.observe.collectionEndpoint.value | toString | trimSuffix "/" }}/v1/kubernetes/v1/entity"
+    logs_endpoint: "{{ .Values.observe.collectionEndpoint.value | toString | trimSuffix "/" }}/v1/kubernetes/v1/entity"
```

Comment:
Do we also need `metrics_endpoint`/`traces_endpoint`?

Reply:
(we don't support traces yet
for metrics, we use a dedicated prometheus endpoint + prometheus remotewrite exporter for it, and it has been running healthily for a while :) https://github.com/observeinc/helm-charts/blob/3c11bc01d8b181a5d86da943e5b3c104e6ea198b/charts/agent/templates/_config-exporters.tpl#L37

The prom remote write exporter does NOT do these "url suffixing" stuff :D